### PR TITLE
Ana/add kava tokens

### LINKED ISSUE #4358: Alt-token fiat currencies values
Probably a very simple bug.

Currently fiat values for alt tokens (currencies other than staking ones) are not being displayed. Also happening in production.

image

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #4358 

Supersedes #4359 

**Description:**

Adds the Kava new tokens! 

Everything working as expected:

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5ef489675e373e0585ed7e9d--lunieio.netlify.app

**Comment by Bitcoinera** [comment]
Ah, I think something was changed in the E2E for the validator list... that's why it is failing again

**Comment by faboweb** [comment]
why is USDX commented out?

**Comment by Bitcoinera** [comment]
> why is USDX commented out?

Because CoinGecko doesn't have it yet, so it was erroring. Should I just remove it completely? It was just meant as a reminder

File: api/lib/fiatvalues-api.js [hunk 1]
```diff
@@ -37,6 +37,8 @@ const denomToCoinGeckoIDDictionary = {
   KRT: 'terra-krw',
   SDT: 'terra-sdt',
   KAVA: 'kava',
+  BNB: 'binancecoin',
+  // USDX: 'usdx',
```
Comment (faboweb) [inline api/lib/fiatvalues-api.js:41]:
```suggestion
  // USDX: 'usdx', // not on coin gecko yet
```

File: api/lib/fiatvalues-api.js [hunk 2]
```diff
@@ -46,6 +48,8 @@ const geckoIDToDenomDictionary = {
   'terra-krw': 'KRT',
   'terra-sdt': 'SDT',
   kava: 'KAVA',
+  binancecoin: 'BNB',
+  // usdx: 'USDX',
```
Comment (faboweb) [inline api/lib/fiatvalues-api.js:52]:
```suggestion
  // usdx: 'USDX', // not on coin gecko yet
```

File: api/lib/fiatvalues-api.js [hunk 3]
```diff
@@ -77,7 +77,7 @@ class fiatValueAPI {
     this.fiatValuesAPIReverseDictionary = geckoIDToDenomDictionary
     this.fiatCurrenciesSymbolsDictionary = fiatCurrenciesSymbolsDictionary
     this.coins = ['ATOM', 'LUNA', 'KRT', 'SDT', 'KAVA', 'BNB', 'KSM'] // Lunie coins currently being traded in the open
-    this.priceFeed = []
+    this.priceFeed = {}
```
Comment (Bitcoinera) [inline api/lib/fiatvalues-api.js:80]: oh, missed that one. Thank you

Comment (Bitcoinera) [reply api/lib/fiatvalues-api.js:80]: I think that was actually in the CoinGecko one 🤔 but yeah

**Pull request merged**